scratch/camera_feed/display_camera_feed.cpp

```diff
@@ -0,0 +1,165 @@
+#include "../../src/network/network.hpp"
+#include "display_camera_feed.hpp"
+#include <stdlib.h>
+#include <string.h>
+#include <iostream>
+#include <turbojpeg.h>
+#include "SDL2/SDL.h"
+//#include <inttypes.h> to print uints
+
+#define CAMERA_MESSAGE_FRAME_DATA_MAX_SIZE 65000
+#define CAMERA_FRAME_BUFFER_SIZE 6220800
+#define CAMERA_FRAME_BUFFER_COUNT 5
+#define WINDOW_X_LENGTH 1920 //window resolution. Standard is 1920
+#define WINDOW_Y_LENGTH 1080 //my window resolution. Standard is 1080
+
+const long unsigned int _jpegSize = WINDOW_X_LENGTH * WINDOW_Y_LENGTH * 3;
+
+void decompress(unsigned char* buffer, unsigned char* compressed_image, int pitch){
+    tjhandle _jpegDecompressor = tjInitDecompress();
+    if(_jpegDecompressor == NULL)
+        std::cout << "decompressor was not made" << std::endl;
+    /*
+        The parameters are the decompressor, the compressed image, the size of the
+        image as an unsigned long, the buffer, the width of the jpeg, the pitch,
+        the height, the pixel type (We have RGB here each contained in 1 byte),
+        and the flags. Basically we say decompress it quickly.
+    */
+    //std::cout << ". JPG size: " << _jpegSize <<  ". Pitch " << pitch << std::endl;
+    if (tjDecompress2(_jpegDecompressor, compressed_image, _jpegSize, buffer, WINDOW_X_LENGTH, pitch, WINDOW_Y_LENGTH, TJPF_RGB, TJFLAG_FASTDCT) == -1)
+        std::cout << "decompressed incorrectly" << std::endl;
+    //free the decompressor.
+    tjDestroy(_jpegDecompressor);
+}
+
+int main(){
+    int pitch = 1920*3;
+    uint32_t offset;
+    //create a buffer for the message struct (declared in loop)
+    unsigned char* temp_compressed_message_buffer = new unsigned char[CAMERA_FRAME_BUFFER_SIZE];
+    //array of buffers to hold the compressed messages
+    BufferItem* frame_buffer = new BufferItem[CAMERA_FRAME_BUFFER_COUNT];
+    //pointer to where we keep the pixel data
+    unsigned char* pixel_location;
+    for(int i = 0; i < CAMERA_FRAME_BUFFER_COUNT; i++){
+        frame_buffer[i].data_sections = new unsigned char[CAMERA_FRAME_BUFFER_SIZE];
+    }
+    uint8_t indx = 0;
+    network::Connection conn;
+    const char* destination_port = "127.0.0.1";
```

Comment:
This is an address, not a port, so this variable should be renamed to reflect that. Then it should be moved to global scope and `CAPITALIZED` since its a constant.